source/billing.txt

```diff
@@ -120,35 +121,61 @@ There are several types of requests:
   For more information on sync operations, see :ref:`Atlas Device Sync Protocol
   <sync-protocol>`.
 
-All incoming requests count toward your monthly usage except for the
+All incoming requests count toward your daily usage except for the
 following:
 
 - Requests to an authentication provider
 - Requests blocked by an application's :ref:`IP Access List <manage-ip-access>`
 - Requests to an invalid HTTP endpoint
 
-**Price:** $2.00 / 1,000,000 requests ($0.000002 / request)
+.. _request-tier-pricing:
 
-**Formula:** (Function Executions + Trigger Executions + GraphQL/Webhook/HTTPS Endpoint Requests + Sync Updates) * $0.000002
+**Price:** 
 
-**Free Tier Threshold:** 1,000,000 :ref:`requests <billing-requests>` or 500
-hours of :ref:`compute <billing-compute>` or 10,000 hours of :ref:`sync
-<billing-sync>` runtime (whichever occurs first)
+.. list-table::
+   :header-rows: 1
+   :widths: 25 25 25 25
+
+   * - Billing Tier
+     - Request Volume (Requests/Day)
+     - Price (per Million Requests)
+     - Price (per Request)
```

Comment:
In the "Introducing Tiered Pricing for App Service Requests" section of the PD:
https://docs.google.com/document/d/1umiArOH7e-X1ve07CiV7lPOKTf_O-QeikugIQH-03gk/edit

Reply:
Can we revert this billing tier section. We were not approved for billing tiers, so that was not implemented as art of the project. Will drop a comment on the PD as well in case other reference it in the future

Reply:
Just reverted, thanks!